### ISSUE TITLE
Assignment 5 Question 1, c, ii

### ISSUE THREAD

**Issue by batubb** [mantasu/cs224n#3]
Can you please explain how you reached the following conclusion in your answer:

Since $\mu_i^{\top}\mu_i=1$, $\mathbf{k}_a$ varies between $(\alpha + 0.5)\mu_a$ and $(\alpha + 1.5)\mu_a$.

**Comment by mantasu** [comment]
Yeah, the explanation doesn't make sense, I don't really remember how I arrived to this. I think the emphasis should be put on the magnitude variance. I'll revise this when I have more time. Thanks for pointing this out!

**Comment by batubb** [comment]
Thank you! I think it makes sense intuitively, I just couldn't derive it mathematically.
